scripts/build_python_sdk.sh

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env bash
+#
+# Builds source and binary Python SDK distributions.
+#
+# Inputs:  ./sdk/python sources generated by the SDK generator; PYPI_VERSION env var set.
+# Outputs: ./sdk/python/bin/dist/{*.tar.gz,*.whl} packages
+
+set -euo pipefail
+
+cd ./sdk/python
+
+# Removing auto-generated setup.py; this is needed until there is an option on the SDK generator to
+# opt out of generating this file.
+rm setup.py
+
+cat << EOF > go.mod
+module fake_python_module // Exclude this directory from Go tools
+
+go 1.17
+EOF
+
+rm -rf ./bin/ ../python.bin/
+cp -R . ../python.bin
+mv ../python.bin ./bin
+
+sed -i.bak -e "s/^  version = .*/  version = \"$PYPI_VERSION\"/g" ./bin/pyproject.toml
```

Comment:
Could we make `PYPI_VERSION` an explicit argument to the script and fail if it's not set?

Reply:
Done.